pkg/services/sqlstore/migrations/ualert/alert_rule.go

```diff
@@ -198,8 +196,6 @@ func transExecErr(s string) (string, error) {
 	switch s {
 	case "", "alerting":
 		return "Alerting", nil
-	case "KeepLastState":
```

Comment:
Same as with transNoData. But additionally, I wonder if the existing case statement is a mistake and it should be like transNoData.

Reply:
Yes checked, so should be "keep_state" like above for the case statement, not the return.

Reply:
and currently migration throws:

`EROR[05-17|15:06:11] Executing migration failed               logger=migrator id="move dashboard alerts to unified alerting" error="unrecognized Execution Error setting keep_state"`

Reply:
@kylebrandt please double check that these are correct, now? Nice catch

Reply:
Yup LGTM, you could make it less lines but adding it to the above line, e.g. `case "", "alerting", "keep_state": return "Alerting", nil` ... but don't care :-)